src/tests/ftest/interoperability/diff_versions.yaml

```diff
@@ -0,0 +1,47 @@
+hosts:
+  test_servers: 3
+  test_clients: 1
+timeout: 950
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  servers:
+    0:
+      targets: 1
+      log_mask: DEBUG
+      env_vars:
+        - DD_MASK=mgmt,md,dsms,any
+        - D_LOG_FLUSH=DEBUG
+pool:
+  control_method: dmg
+  scm_size: 5G
+  name: daos_server
+container:
+  control_method: daos
+  type: POSIX
+  properties: rf:0
+attrtests:
+  num_attributes: 20
+dfuse:
+  mount_dir: "/tmp/daos_dfuse1/"
```

Comment:
FYI that dfuse_utils will now generate a mount dir if a test doesn't specify:
`/tmp/daos_dfuse_<test_name>`

Reply:
Thanks for the information.